### PR TITLE
1.11.0

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Change Log
+## 1.11.0 - 2019-03-22
+### Added
+* Add IoT Edge marketplace. User can view and create Edge modules from marketplace.
```
Comment (adashen) [inline CHANGELOG.md]: Azure IoT Edge modules?

Comment (SLdragon) [reply CHANGELOG.md]: Oh, OK

Comment (formulahendry) [inline CHANGELOG.md]: from Azure Marketplace

Comment (SLdragon) [reply CHANGELOG.md]: There so many Azure inside, do we really need this? :joy:
Add **Azure** IoT Edge marketplace. User can view and create **Azure** IoT Edge modules from **Azure** Marketplace

Comment (SLdragon) [reply CHANGELOG.md]: Or just :
Add Azure IoT Edge marketplace. User can view and create Azure IoT Edge modules from it

Comment (adashen) [reply CHANGELOG.md]: Upper case for "Marketplace"?

Comment (formulahendry) [reply CHANGELOG.md]: How about `View and import Azure IoT Edge modules from Azure Marketplace`?

File: README.md [hunk 2]
```diff
@@ -15,10 +15,10 @@
 
 ![Added](images/multipleplatform.gif)
 
-## What's New (v1.10.0)
+## What's New (v1.11.0)
 ### Added
-* Add sample gallery page. User can view and create Azure IoT Edge solution based on samples.
-* Add issue template
+* Add IoT Edge marketplace. User can view and create Edge modules from marketplace.
```
Comment (xinyiz1226) [inline README.md]: Please make sure the wording here is consistent with the ones in changelog after modification. Thank you.

**Comment by SLdragon** [comment]
Any suggestions about this:

Add IoT Edge Marketplace page. User can view and create IoT Edge modules from Azure Marketplace.

@formulahendry @xinyiz1226 @adashen

**Pull request merged**